shared/src/commonMain/kotlin/com/example/myplants/core/data/ColumnAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.myplants.core.data
+
+import com.squareup.sqldelight.ColumnAdapter
+
+val listOfStringsAdapter = object : ColumnAdapter<List<String>, String> {
+    override fun decode(databaseValue: String) =
+        if (databaseValue.isEmpty()) {
+            listOf()
+        } else {
+            databaseValue.split(",")
```

Comment:
This works as long as the strings don't contain any commas, better use JSON encoding